locales/pt_BR.json

```diff
@@ -0,0 +1,777 @@
+			"pass": "O item da lista de descrição tem um elemento pai <dl>",
+			"fail": "O item da lista de descrição não tem um elemente pai <dl>"
+		},
+		"listitem": {
+			"pass": "O item de lista tem um elemento pai <ul>, <ol> ou role=\"list\"",
+			"fail": "O item de lista não tem um elemento pai <ul>, <ol> ou role=\"list\""
+		},
+		"only-dlitems": {
+			"pass": "O elemento de lista tem somente filhos diretos que são permitidos dentro de elementos <dt> ou <dd>",
+			"fail": "O elemento de lista tem filhos diretos que não são permitidos dentro de elementos <dt> ou <dd>"
+		},
+		"only-listitems": {
+			"pass": "O elemento de lista tem somente filhos diretos que são permitidos dentro de elementos <li>",
+			"fail": "O elemento de lista tem filhos diretos que não são permitidos dentro de elemento <li>"
+		},
+		"structured-dlitems": {
+			"pass": "Quando não vazio, o elemento tem ambos elementos <dt> e <dd>",
+			"fail": "Quando não vazio, o elemento não tem nem ao menos um elemento <dt> seguindo de pelo menos um elemento <dd>"
+		},
+		"caption": {
+			"pass": "O elemento multimídia tem uma trilha de legendas",
+			"incomplete": "Verifique se legendas estão disponíveis para o elemento"
+		},
+		"description": {
+			"pass": "O elemento multimídia tem uma trilha de audiodescrição",
+			"incomplete": "Verifique se a audiodescrição está disponível para o elemento"
+		},
+		"frame-tested": {
+			"pass": "O 'iframe' foi testado com o 'axe-core'",
+			"fail": "O 'iframe' não pôde ser testado com o 'axe-core'",
+			"incomplete": "O 'iframe' ainda tem que ser testado com o 'axe-core'"
+		},
+		"css-orientation-lock": {
+			"pass": "A exibição é operável, e o bloqueio da orientação não existe",
+			"fail": "Bloqueio de orientação CSS aplicado, o que torna a exibição inoperável",
+			"incomplete": "Bloqueio de orientação CSS não pode ser determinado"
+		},
+		"meta-viewport-large": {
+			"pass": "A tag <meta> não impede o zoom significativo em dispositivos móveis",
+			"fail": "A tag <meta> limita o zoom em dispositivos móveis"
+		},
+		"meta-viewport": {
+			"pass": "A tag <meta> não desabilita o zoom em dispositivos móveis",
+			"fail": "{{=it.data}} na tag <meta> desabilita o zoom em dispositivos móveis"
+		},
+		"header-present": {
+			"pass": "A página tem um cabeçalho (header)",
+			"fail": "A página não tem um cabeçalho (header)"
+		},
+		"heading-order": {
+			"pass": "Hierarquia de títulos válida",
+			"fail": "Hierarquia de títulos inválida"
+		},
+		"internal-link-present": {
+			"pass": "Link de escape válido encontrado",
+			"fail": "Nenhum link de escape válido encontrado"
+		},
+		"landmark": {
+			"pass": "A página tem uma região (landmark)",
+			"fail": "A página não tem nenhuma região (landmark)"
+		},
+		"meta-refresh": {
+			"pass": "A tag <meta> não atualiza a página imediatamente",
+			"fail": "A tag <meta> força atualizações temporizadas da página"
+		},
+		"p-as-heading": {
+			"pass": "Elementos <p> não são estilizados como títulos",
```

Comment:
The WCAG PT_BR translation for header is "cabeçalho", not "título"

Reply:
As discussed before.

Reply:
And the word in English used here is 'heading' and not 'header'.